code/modules/1713/jobs/civilian.dm

```diff
@@ -178,6 +161,62 @@
 	min_positions = 1
 	max_positions = 2
 
+/datum/job/civilian/banker
+	title = "Banker"
+	en_meaning = "Finance Minister"
+	rank_abbreviation = "Banker"
+	can_be_female = TRUE
+	spawn_location = "JoinLateCivB"
+	is_merchant = TRUE
+	is_officer = TRUE
+	is_commander = TRUE
+
+	whitelisted = TRUE
+	is_1713 = TRUE
+
+	min_positions = 1
+	max_positions = 1
+
+/datum/job/civilian/banker/equip(var/mob/living/human/H)
```

Comment:
I think this is duplicated earlier in the code, so you may need to remove the previous datum

Reply:
Yeah, forgot to remove it. Fixed